src/main/scala/com/intersys/mdc/challenge/exercises/problems/Problem3.scala

```diff
@@ -4,6 +4,28 @@ import akka.http.scaladsl.server.Route
 
 case object Problem3 extends Problem {
 
+  def factorial(n: Long): Long = n match {
+    case 0 => 1
+    case _ => n * factorial(n-1)
+  }
+
+  def tPascal(depth:Int,row:Int,column:Int,triangulo:String): String = {
+
+    if(column < row && row < depth)
+      triangulo + (factorial(row) / (factorial(column) * factorial(row-column))) + " " + tPascal(depth,row,column+1,triangulo)
```

Comment:
👎 Could have avoid code repetition by adding a routine/procedure (Not so clean/DRY).